Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/azure/webapp/deploymentslot/DeploymentSlotModule.java

```diff
@@ -25,9 +28,15 @@ public DeploymentSlotModule(final Node parent, final String subscriptionId, fina
     @Override
     protected void refreshItems() throws AzureCmdException {
         try {
-            presenter.onModuleRefresh(this.subscriptionId, this.webAppId);
+            presenter.onRefreshDeploymentSlotModule(this.subscriptionId, this.webAppId);
         } catch(Exception ex) {
             ex.printStackTrace();
         }
     }
+
+    @Override
+    public void render(@NotNull final List<DeploymentSlot> slots) {
```

Comment:
let's use a more explicit name, what do you think of `renderDeploymentSlots`/`renderChildren`.